### PR TITLE
#217 fix position_ids

### PR DESCRIPTION
**Pull request by hijkzzz**
see #217

### CODE REVIEW

File: openrlhf/models/actor.py [hunk 1]
```diff
@@ -175,7 +175,10 @@ def forward(
         return_output=False,
     ) -> torch.Tensor:
         """Returns action log probs"""
-        output = self.model(sequences, attention_mask=attention_mask)
+        # https://github.com/OpenLLMAI/OpenRLHF/issues/217
+        position_ids = attention_mask.long().cumsum(-1) - 1
+        position_ids.masked_fill_(attention_mask == 0, 1)
```
[the remark this answers is not in the document]
Comment (hijkzzz) [reply openrlhf/models/actor.py:180]: copied from https://github.com/huggingface/transformers/blob/1d0ea7abe0a3d73414545fd401742ad25b0e7795/src/transformers/models/llama/modeling_llama.py#L1250

**Comment by karthik-nexusflow** [comment]
we need to test this further , initial experiments with this change is giving weird policy loss
